front/src/views/user/history-previous-trips/history-previous-trips.view.html

```diff
@@ -0,0 +1,37 @@
+<div>
+  <div class="d-flex flex-wrap align-stretch justify-space-between">
+    <div>
+      <h4 class="accent--text text--lighten-2">{{ $t('tripHistory.previousTrips') }}</h4>
+    </div>
+    <div>
+      <mmt-text-input class="search-text-field"
+                      :placeholder="$t('tripHistory.search')"
+                      mmt-label=" "
+                      rounded
+                      filled
+                      dense
+                      append-icon="mdi-find-replace"
+                      @click:append="getTrips"
+                      v-model="searchQuery" />
+    </div>
+  </div>
+  <div class="d-flex flex-wrap justify-space-between my-5">
+    <h5 v-for="y in years" :key="y">
+      <a v-on:click="filterByYear(y)"
+         :class="year === y ? `primary--text` : `accent--text text--lighten-2`">
+        {{y}}
+      </a>
```

Comment:
it is better to communicate about flow and design with designers because for me it is strange that we request trips by years. I think it is only dividers. And searching as for me is provided by all trips, I don`t see some filter for searching by year on mockups.

Reply:
I have communicated with designers. Years can use for for filtering.